src/main/java/training/commands/ArmCommands/elbowCommands/ElbowDefultCommand.java

```diff
@@ -0,0 +1,24 @@
+package training.commands.ArmCommands.elbowCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.ArmSubsystems.elbowSubsystem.Elbow;
+
+public class ElbowDefultCommand extends Command {
+    private Elbow elbow;
+
+    public ElbowDefultCommand() {
+        elbow = Elbow.getInstance();
+    }
+
+    @Override
+    public void initialize() {
+        elbow.stayAtPosition();
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        super.end(true);
```

Comment:
```suggestion
        super.end(interrupted);
```
my bad, no need for this, interrupted is the default